src/sentry/static/sentry/app/components/events/eventAttachments.jsx

```diff
@@ -2,18 +2,47 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {Box} from 'grid-emotion';
 
+import Button from 'app/components/button';
 import FileSize from 'app/components/fileSize';
 
 import {t} from 'app/locale';
 import {Panel, PanelBody, PanelItem} from 'app/components/panels';
+import SentryTypes from 'app/sentryTypes';
 import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import ConfigStore from 'app/stores/configStore';
+import MemberListStore from 'app/stores/memberListStore';
+
+function isRoleFulfilled(availableRoles, requiredRole) {
+  const user = ConfigStore.get('user');
+  if (!user) {
+    return false;
+  }
+
+  if (user.isSuperuser) {
+    return true;
+  }
+
+  if (!Array.isArray(availableRoles)) {
+    return false;
+  }
+
+  const member = MemberListStore.getById(user.id);
+  const currentRole = member && member.role;
+
+  const roleIds = availableRoles.map(role => role.id);
+  const requiredIndex = roleIds.indexOf(requiredRole);
+  const currentIndex = roleIds.indexOf(currentRole);
+  return currentIndex >= requiredIndex;
```

Comment:
Are roles guaranteed to be sorted like this? Is it possible for both required and current indexes to be `-1`?

Reply:
Correct, roles are sorted by ascending priority (or actually, priority is the index in the list). Only with invalid inputs either could be `-1`, but the API should have validated this before.